src/main/resources/org/jenkinsci/plugins/authorizeproject/strategy/SystemAuthorizationStrategy/config.jelly

```diff
@@ -0,0 +1,55 @@
+<!--
+The MIT License
+
+Copyright (c) 2016 Stephen Connolly
```

Comment:
Yep, that code was written in my spare time... but the splitting of #13 was on company time